### PR TITLE
Remove support for legacy private links

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

**Comment by dschmidt** [comment]
Which was the last ownCloud Server version that required this?

File: src/gui/accountsettings.cpp [hunk 1]
```diff
@@ -263,7 +263,7 @@ void AccountSettings::slotCustomContextMenuRequested(const QPoint &pos)
         QString path = info->_folder->remotePathTrailingSlash();
         path += info->_path;
         connect(ac, &QAction::triggered, this, [this, path]{
-            fetchPrivateLinkUrl(_accountState->account(), path, {}, this, [](const QString &url) {
```
Comment (dschmidt) [inline src/gui/accountsettings.cpp:266]: Uargh 🙈

**Review by dschmidt — APPROVED** [review]

**Comment by TheOneRing** [comment]
This will go to master now, in 2.8 we will officially remove support for all servers < 10

**Comment by dschmidt** [comment]
Sounds good!

**Pull request merged**